File: packages/java/src/main/java/com/tesote/sdk/errors/AccountNotFoundException.java
package com.tesote.sdk.errors;

public final class AccountNotFoundException extends NotFoundException {
    public AccountNotFoundException(String message, String errorCode, int httpStatus,
                                    String requestId, String errorId, Integer retryAfter,
                                    String responseBody, RequestSummary requestSummary,
                                    int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/AccountNotFoundException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>404 ACCOUNT_NOT_FOUND.</summary>
public sealed class AccountNotFoundException : NotFoundException
{
    /// <summary>Construct with the full required-field set.</summary>
    public AccountNotFoundException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}
